manifests/application.pp

```diff
@@ -18,18 +18,6 @@
   include certs::pulp_client
   include katello::params
 
-  foreman_config_entry { 'pulp_client_cert':
```

Comment:
If we don't set these, how does the application still know about them?

Reply:
Checkout https://github.com/Katello/katello/pull/9330 -- we are dropping the settings and handling the paths there.